### DISCUSSION TITLE
Pushover Publisher including a target variable

### DISCUSSION THREAD

**Discussion by BambamNZ** [netalertx/NetAlertX#588] [Q&A]
Would it be worthwhile considering including a "target" or "device" variable in the Pushover configuration section?
This would give the ability to push messages to the selected device and not every device configured in Pushover dashboard.
The device parameter is optional, but will give flexibility to those with multiple devices in a single Pushover account.

**Comment by jokob-sk** [comment]
Hi @BambamNZ ,

Pushover was implemented by @Schlump so let's see if he has time to have a look at your request. I don't use pushover so unsure how exactly the use case you describe would work. 

Happy for you to have a look at the implementation and submit a PR as well. Please keep your changes within the plugin if possible. 

j

**Reply by BambamNZ** [reply]
Thanks for the reply

Would love to help, however my knowledge of Python is pretty limited, can deduct what I think may be required but, will be missing some details surely. It is adding a device field on the UI and then passing that to the call to Pushover, if the device field contains a name of a device, the message will be passed to that device in Pushover, not a require field but very helpful.

**Reply by jokob-sk** [reply]
Hey @BambamNZ ,

The original plugin author, @Schlump, had some time and submitted a PR request so you can test the feature in the `pi.alert_dev` image in about 10 min from now.

**Reply by BambamNZ** [reply]
Hi

Had a issue with Environment variables and the local TZ causing errors - sorted by removing TZ as a variable.  

Very much appreciated, looking into this.

Pushover testing, works perfectly, once a device is entered the alerts only goes to the defined device. 
No device defined sends message to all devices - expected behavior 

Looking forward see it in the main release

**Reply by jokob-sk** [reply]
Thanks for checking!

FYI this is released
